Week1/homework/app.js

```diff
@@ -1,11 +1,138 @@
 'use strict';
 
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+  // Q1.1
+
+  const books = [
+    'animal_farm',
+    'veronica_decide_to_die',
+    '1984',
+    'the_lord_of_the_rings',
+    'the_book_thief',
+    'romeo_and_juliet',
+    'the_color_purple',
+    'alice_in_wonderland',
+    'frankenstein',
+    'the_kite_runner',
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  // Q1.3
+
+  function booksGenerator() {
+    const bookList = document.createElement('ul');
+    const main = document.getElementById('main');
+    main.appendChild(bookList);
+    for (let i = 0; i < books.length; i++) {
+      const Li = document.createElement('li');
+      Li.innerText = books[i];
+      bookList.appendChild(Li);
+    }
+  }
+  booksGenerator();
+}
+
+// Q1.4
+
+const booksInformation = {
+  animal_farm: {
+    title: 'animal farm',
+    language: 'english',
+    author: 'george orwell',
+  },
+  veronica_decide_to_die: {
+    title: 'veronica decides to die',
+    language: 'english',
+    author: 'Paulo Coelho',
+  },
+  1984: {
+    title: '1984',
+    language: 'english',
+    author: 'george orwell',
+  },
+  the_lord_of_the_rings: {
+    title: 'the lord of the ring',
+    language: 'english',
+    author: ' j.r.r. tolkien',
+  },
+  the_book_thief: {
+    title: 'the book thief',
+    language: 'english',
+    author: 'markus zusak',
+  },
+  romeo_and_juliet: {
+    title: 'romeo and juliet',
+    language: 'english',
+    author: 'william shakespeare',
+  },
+  the_color_purple: {
+    title: 'the color purple',
+    language: 'english',
+    author: 'alice walker',
+  },
+  alice_in_wonderland: {
+    title: 'alice in wonderland',
+    language: 'english',
+    author: 'lewis carroll',
+  },
+  frankenstein: {
+    title: 'frankenstein',
+    language: 'english',
+    author: 'mary sheley',
+  },
+  the_kite_runner: {
+    title: 'the kite runner',
+    language: 'english',
+    author: 'khaled hosseini',
+  },
+};
+
+// Q1.5
+
+function getInformation() {
```

Comment:
This function name doesn't reflect what it's doing. Instead, use a name like "renderBookDetails" or "displayBookList".